pkg/varlinkapi/util.go

```diff
@@ -191,7 +191,7 @@ func makePsOpts(inOpts iopodman.PsOpts) shared.PsOptions {
 		Latest:    derefBool(inOpts.Latest),
 		NoTrunc:   derefBool(inOpts.NoTrunc),
 		Pod:       derefBool(inOpts.Pod),
-		Size:      true,
+		Size:      derefBool(inOpts.Sync),
```

Comment:
shouldn't this be `derefBool(inOpts.Sort)`?

Reply:
fixed. Should be derefBool(inOpts.Size)

Reply:
nice catch @giuseppe, I missed that.